Dockerfile

```diff
@@ -36,54 +33,43 @@ RUN if [ "${BUILD_DATE}" = "undefined" ]; then \
   npm run build -- --base=$UI_PUBLIC_URL
 
 ### Build API ###
-FROM --platform=${BUILDPLATFORM:-linux/amd64} golang:1.22-bookworm AS api
+FROM --platform=${BUILDPLATFORM:-linux/amd64} golang:1.22-alpine AS api
 ARG TARGETPLATFORM
 
-# See for details: https://github.com/hadolint/hadolint/wiki/DL4006
-SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
-
 WORKDIR /app/api
 
 ENV GOPATH="/go"
 ENV PATH="${GOPATH}/bin:${PATH}"
 ENV CGO_ENABLED=1
 
 # Download dependencies
-COPY scripts/apt/debian-testing.sources /etc/apt/sources.list.d/
-COPY scripts/set_compiler_env.sh /app/scripts/
 COPY scripts/install_build_dependencies.sh /app/scripts/
 COPY scripts/install_runtime_dependencies.sh /app/scripts/
 RUN chmod +x /app/scripts/*.sh \
   && /app/scripts/install_build_dependencies.sh \
+  && mv $(go env GOPATH)/bin/reflex /usr/bin/ \
   && /app/scripts/install_runtime_dependencies.sh
 
 COPY api/go.mod api/go.sum /app/api/
-RUN source /app/scripts/set_compiler_env.sh \
-  && go env \
+RUN go env \
   && go mod download \
-  # Patch go-face
-  && sed -i 's/-march=native//g' ${GOPATH}/pkg/mod/github.com/!kagami/go-face*/face.go \
   # Build dependencies that use CGO
   && go install \
     github.com/mattn/go-sqlite3 \
     github.com/Kagami/go-face
 
 COPY api /app/api
-RUN source /app/scripts/set_compiler_env.sh \
-  && go build -v -o photoview .
+RUN go build -v -o photoview .
 
 ### Build release image ###
-FROM --platform=${BUILDPLATFORM:-linux/amd64} debian:testing-slim AS release
+FROM --platform=${BUILDPLATFORM:-linux/amd64} alpine:latest AS release
 ARG TARGETPLATFORM
 
-# See for details: https://github.com/hadolint/hadolint/wiki/DL4006
-SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
-
 COPY scripts/install_runtime_dependencies.sh /app/scripts/
 RUN chmod +x /app/scripts/install_runtime_dependencies.sh \
   # Create a user to run Photoview server
-  && groupadd -g 999 photoview \
-  && useradd -r -u 999 -g photoview -m photoview \
+  && addgroup -g 9999 photoview \
+  && adduser -u 9999 -G photoview -D photoview \
```

Comment:
I'd agree that we have to fix this ID by setting it from the unprivileged range, but this will be a breaking change for users, who already created a user and group on their host to match the IDs in the container. Is it unavoidable (like, the 999 is busy)?

Reply:
OK. Reverted.

Reply:
Unfortunately, `alpine` reserves `999`.

```
 > [linux/amd64->arm/v6 release 3/7] RUN chmod +x /app/scripts/install_runtime_dependencies.sh   && addgroup -g 999 photoview   && adduser -u 999 -G photoview -D photoview   && /app/scripts/install_runtime_dependencies.sh:
0.130 addgroup: gid '999' in use
------
------
 > [linux/amd64->arm/v7 release 3/7] RUN chmod +x /app/scripts/install_runtime_dependencies.sh   && addgroup -g 999 photoview   && adduser -u 999 -G photoview -D photoview   && /app/scripts/install_runtime_dependencies.sh:
0.169 addgroup: gid '999' in use
------
------
 > [linux/amd64->arm64 release 3/7] RUN chmod +x /app/scripts/install_runtime_dependencies.sh   && addgroup -g 999 photoview   && adduser -u 999 -G photoview -D photoview   && /app/scripts/install_runtime_dependencies.sh:
0.129 addgroup: gid '999' in use
```

Reply:
Ok, then this is the time to introduce this breaking change.
This means that the next version would be again major

Reply:
Done.